### PR TITLE
[WEB-1697] Persist selected clinic ID when using the browser navigation buttons

### PR DESCRIPTION
**Pull request by clintonium-119**
See [WEB-1697]

[WEB-1697]: https://tidepool.atlassian.net/browse/WEB-1697?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by clintonium-119** [comment]
@krystophv Can you also have a look at  [this commit](https://github.com/tidepool-org/blip/pull/1082/commits/e01ebd851d8666ecbfd37224d8855dfd8353beab)?  It's a small change based on an edge case brian found upon logins that go directly to the clinic workspace.

File: test/unit/redux/actions/async.test.js [hunk 1]
```diff
@@ -541,7 +541,7 @@ describe('Actions', () => {
       });
     });
 
-    describe('login', () => {
+    describe.only('login', () => {
```
Comment (krystophv) [inline test/unit/redux/actions/async.test.js:544]:
```suggestion
    describe('login', () => {
```

Comment (clintonium-119) [reply test/unit/redux/actions/async.test.js:544]: Ha :)  Thanks.  Good catch

**Review by krystophv — APPROVED** [review]
Updates LGTM :shipit:
